gtk-update-icon-cache is only needed here when installing locally
and gets skipped when installing to a destdir

--- a/meson.build
+++ b/meson.build
@@ -45,14 +45,3 @@ else
   )
 endif
 
-gtk_update_icon_cache = find_program(
-  'gtk4-update-icon-cache',
-  'gtk-update-icon-cache',
-  required : true,
-)
-meson.add_install_script(
-  gtk_update_icon_cache,
-  '-qtf',
-  get_option('prefix') / adwaita_dir,
-  skip_if_destdir: true,
-)
